src/HtmlUtils.js

```diff
@@ -84,7 +86,8 @@ export function isSingleEmoji(str) {
  * @return {String} The shortcode (such as :thumbup:)
  */
 export function unicodeToShortcode(char) {
-    const data = EMOJIBASE.find(e => e.unicode === char);
+    const emptyVariation = char + VARIATION_SELECTOR;
```

Comment:
Please add a comment to explain why this is needed. It would also be good link to the related issue in the comment as well.

Reply:
Great, thanks! 😁